### ISSUE TITLE
Remove Brave

### ISSUE THREAD

**Issue by Shifterovich** [privacytoolsIO/privacytools.io#161]
Most people want Brave removed and only a few wanted to add it, should we remove it?

**Comment by Hillside502** [comment]
@Shifterovich 
Who are "most people"?
Where is the discussion?

**Comment by Hillside502** [comment]
#78

**Comment by Shifterovich** [comment]
https://www.reddit.com/r/privacytoolsIO/comments/5g160q/new_browser_recommendation_brave_discussion_thread/

https://www.reddit.com/r/privacytoolsIO/comments/5nt0nj/curious_why_brave_is_on_the_recommended_browser/

And comments in a lot of threads.

**Comment by Hillside502** [comment]
@Shifterovich 
Thanks for the reddit links.  Pros and Cons abound.

Interesting to see what thinking develops in the privacytools.io and PRISM Break communities.

**Comment by fuelflo** [comment]
+1

I also don't understand why brave is in the list. Just read their privacy info on their site (way down next to the terms of service).

When the ad blocking features are enabled (which is the default) the can inject their own ads and collect data....thats not really a privacy tool!

> >  Tracking & Ads
> 
> Brave and our advertising partners may provide ads to you through Brave when you have enabled the Ad Replacement feature (which is the default setting in any of our browsers). These Ads do not use tracking technologies like cookies or any other persistent storage on your browser. Browsing history and other information that is used to generate intent signals is encrypted using a client generated key such that Brave cannot see or use this information for any purposes other than Ad Replacement. The encrypted data is stored by Brave for the sole purpose of syncing across the User’s devices that utilize a Brave browser, but Brave does not have access to the client generated key that can be used to decrypt the data and therefore cannot “read” or “see” this data. If you choose to disable the Ad Replacement or Ad Blocking features available through Brave’s settings, this may allow third party cookies and persistent storage technologies to collect information about you and your browsing activities. This may include collection of your personally identifiable information, but Brave does not receive or use this information.
> 
> Brave will also collect anonymous ad campaign related data that will be used for reporting and payments to third parties. This includes payments to both ad publishers and Brave partners for Ads viewed by you using Brave Ad Replacement. This data cannot be linked to your device(s) or otherwise identify you. Brave will not collect any information for ads that are delivered when Ad Replacement or Ad Blocking is not enabled.

Doesn't matter its "encrypted" but they are using your browsing history, etc.

**Comment by kewde** [comment]
On a note, there are other reasons why you should be skeptical of Brave (and Electron in general).
http://blog.scottlogic.com/2016/03/09/As-It-Stands-Electron-Security.html

On the other hand, Brave has been a leader in fixing security issues in Electron, so I don't know how to weigh this one off.

**Comment by bakku** [comment]
Interesting article, I never had this in mind when I heard about Electron.
I wonder how things are now since the article is one year old.

**Comment by nnvn** [comment]
I found privacytools.io today. saw brave mentioned and went to wikipedia. Got confused. blocks ads to inject their own ads??

**Comment by Hillside502** [comment]
@nnvn 
"...With the ad-tech ecosystem out of control, users have revolted and blocking ads has become the new weapon of choice for improving their browsing speed, safety and privacy. Unfortunately, blocking alone results in a race to the bottom where nobody wins..."
https://www.brave.com/about.html

**Comment by gjallarhorn** [comment]
I just tried Brave on a Windows machine and on a Android phone and both blocked ads, 3rd party cookies and privacy invading scripts by default. In other words, their default settings did not insert Braves own ads as their policy tells us. (https://www.brave.com/privacy.html)

Their privacy policy have not been updated since March 4, 2016, so it could be the case that this was before they decided how to fund the project.  We still do not know what their future plans are of course, so things can change.

**Comment by bbondy** [comment]
I'm a co-founder for Brave.

We block ads (Similar to uBlock), block tracking (similar to Disconnect), and upgrade HTTPS (Similar to HTTPS Everywhere) by default. Fingerprinting protection can be turned on but is currently disabled by default for web compat. We're working to default it on as well. NoScript is built in as well as other privacy and security measures.  We're currently working on Tor private tabs.

No browsing history is transmitted to Brave in any form.  The only way it can even leave your computer is if you turn on the opt-in, client side encrypted Sync and that is stored on S3. We don't have access to others data, we don't want it, and we made sure we can't change our minds by using client side encryption.

Brave ads only exist as an experimental setting, and it is and will remained disabled by default and opt-in.  If released one day, it is using on device only data and opt-in.

Brave payments (donations to publishers) is opt-in, done anonymously, and on device if enabled using the Anonize protocol. 

Code is fully open source and auditable here:
https://github.com/brave/browser-laptop
https://github.com/brave/muon
https://github.com/brave/ad-block
https://github.com/brave/tracking-protection
https://github.com/brave/sync

**Comment by mrtargaryen** [comment]
@bbondy I've been using Brave for over a year now (i think). However its still not my default browser due to the lack of extensions and with your javascript blocking feature I have no control over what scripts can run or be blocked, thus most sites are broken. Will you be adding extensions like u-matrix? And when can we see a brave addon/extension store?

**Comment by gjallarhorn** [comment]
@mrtargaryen Brave 1.0 will have full extension support. Source:
https://www.brave.com/development-plans-for-upcoming-release/

**Comment by bbondy** [comment]
@mrtargaryen per origin JavaScript blocking is available in the current released product, just turn on NoScript globally or per site, then  there's an icon in the URLbar to customize which origins to apply exceptions to.

We'll be adding native built-in similar controls for other shields. We'll support extensions like that though in the meantime with the brave-core rewrite.

**Comment by mrtargaryen** [comment]
@bbondy ahh I see, I don't know how I missed that. Thanks.

**Comment by mrtargaryen** [comment]
@gjallarhorn thanks for that, I look forward to more extensions. I currently use a hardened Fireox which I like, and unfortunately I have to use Chrome for My university work. And I can see how frequently Chrome"calls home" with Little Snitch. Having a stable chromium based browser will be great. I can finally get rid of everything Google off my Mac.

**Comment by Shifterovich** [comment]
https://github.com/privacytoolsIO/privacytools.io/commit/bd78b527033e0787e95a49d8d66d7ea2b7436208

**Comment by dm17** [comment]
Pretty lame to see #649 closed, when @ciampolo's points were never addressed.

**Comment by quantumpacket** [comment]
I just got spammed by Coinbase advertising this for the Brave browser: https://www.coinbase.com/earn/basic-attention-token Free tokens!? \*clears throat\* If it's free, you're not the customer, you're the product.

**Comment by Shifterovich** [comment]
I don't see anything about free tokens on that page. I see rewards for watching videos.

> If it's free, you're not the customer, you're the product.

This applies to scenarios where you get something for free but are actually paying with your data.

Here you're getting rewarded for watching videos. Assuming no ugly tracking, this is just a marketing campaign from Brave's side.

**Comment by KonoromiHimaries** [comment]
https://twitter.com/tomscott/status/1076160882873380870

**Comment by dm17** [comment]
At this point it is pretty obvious that PTIO is part of a controlled opposition effort to make sure the big players stay on top. If you really believe Brave CEO donated $1K against gay marriage - while seeming gay himself - to get pushed out of Mozilla and start a top "opposition" browser, then I guess you're a sucker! 

No reason at this point that PTIO can't recommend ungoogled-chromium, waterfox, pale moon, and the like. But it recommends none of those - and doesn't waver in the face of evidence.

**Comment by blacklight447-ptio** [comment]
@dm17 palemoon is insecure as it allows unsigned extensions and does not use tab sandboxing. Wtaerfox supports npapi whichvis a security disaster( and also unsigned addons.), and ungoogled chromium has a delay in updates from normal chromium. You makes some pretty wide claims based on nothing but lies and conspiracy. Anyhow, if you dont like the site, you can always start your own with youe own recommendations ;).

**Comment by Mikaela** [comment]
> https://twitter.com/tomscott/status/1076160882873380870

I found that difficult to read, but https://threadreaderapp.com/thread/1076160753793683456.html was more clear and ends with

> A final update on the thread about Brave: they're now opt-in for creators! While it's still possible to tip folks who haven't opted in, the data is stored in-browser and the UI has been clarified. These are good changes, and they fix the complaints I had!

I cannot see a date, but I imagine it has been in January.

> No reason at this point that PTIO can't recommend ungoogled-chromium, waterfox, pale moon, and the like. But it recommends none of those - and doesn't waver in the face of evidence.

Ungoogled-Chromium currently ([when I last heard](https://forum.privacytools.io/t/installing-ungoogled-chromium-on-ubuntu-19-04/317/2?u=mikaela)) requires installing a random package from a random source and will possibly not have automatic updates, so I see it as a security risk. There is a closed issue https://github.com/Eloston/ungoogled-chromium/issues/36 on Flatpak/Snap support which I see as the best method for distributing software/updates across distributions without which I wouldn't support it (and open issue about a PPA that would benefit only Ubuntu users https://github.com/Eloston/ungoogled-chromium/issues/37 and which I think are practically deprecated by Flatpak/Snap). I also don't know how many people are in the team or if it's a bus factor issue like with Waterfox discussed (at least by me) in https://github.com/privacytoolsIO/privacytools.io/issues/856.

**Comment by KonoromiHimaries** [comment]
> Brave collaborates with Facebook. I suggest you see the "source code" of the start page (you click the right mouse button on the start page and you call the page preview) - and how nice FB scripts sit there ...  // https://www.wykop.pl/link/4993027/comment/66316435/#comment-66316435

**Comment by 0ndrey** [comment]
It's useless. PTIO is voting for Brave and is not going to stop recommending this one of the most insecure browsers. It is worth noting that a month ago they introduced Brave user agent into their browser and now its fingerprint is truly unique, accounting for less than half the percent of users worldwide (according to AmIUnique).

> 
> 
> > Brave collaborates with Facebook. I suggest you see the "source code" of the start page (you click the right mouse button on the start page and you call the page preview) - and how nice FB scripts sit there ...  // https://www.wykop.pl/link/4993027/comment/66316435/#comment-66316435
> 
> https://www.stopusingfacebook.co/

It's useless. PTIO is voting for Brave and is not going to stop recommending this one of the most insecure browsers. It is worth noting that a month ago they introduced Brave user agent into their browser and now its fingerprint is truly unique, accounting for less than half the percent of users worldwide (according to AmIUnique).

**Comment by 0ndrey** [comment]
> 
> 
> > Brave collaborates with Facebook. I suggest you see the "source code" of the start page (you click the right mouse button on the start page and you call the page preview) - and how nice FB scripts sit there ...  // https://www.wykop.pl/link/4993027/comment/66316435/#comment-66316435
> 
> https://www.stopusingfacebook.co/

It's useless. PTIO is voting for Brave and is not going to stop recommending this one of the most insecure browsers. It is worth noting that a month ago they introduced Brave user agent into their browser and now its fingerprint is truly unique, accounting for less than half the percent of users worldwide (according to AmIUnique).

**Comment by Shifterovich** [comment]
These discussions are never going to end. @privacytoolsIO/editorial can we make a final decision on Brave?

I personally see only one pro and one con.

Pro: Native ad blocking is more performant than using browser extensions.
Cons: It's more fingerprintable than Chrome. Trying to appear like it protect users against fingerprinting at this stage is dishonest.

**Comment by KonoromiHimaries** [comment]
<details>
  <summary> @Shifterovich </summary>

<details>
  <summary>

> Pro: Native ad blocking is more performant than using browser extensions.

</summary>

uBlock Origin is more than simple adblocker inclided in brave, this is a spectrum adblocker which supports advanced filters, like https://github.com/yourduskquibbles/webannoyances

performance with so many filters is not achievable in brave browser

</details>

<details>
  <summary>

> Con: It's more fingerprintable than Chrome. Trying to appear like it protect users against fingerprinting at this stage is dishonest.

</summary>

still you can use extensions, like https://chrome.google.com/webstore/detail/canvas-blocker-fingerprin/nomnklagbgmgghhjidfhnoelnjfndfpd

</details>

</details>

**Comment by beerisgood** [comment]
@KonoromiHimaries I agree that uBlock Origin is way better, but blocking canvas with add-on / disable it makes you more unique.
Better using Firefox internal "resist fingerprinting" feature which use the Tor browser canvas fingerprint

**Comment by Shifterovich** [comment]
> still you can use extensions

The browser is marketed as fingerprinting resistant as-is, without needing any extensions. Not to mention that installing an extension like you mentioned would likely give you a much more unique fingerprint because you could be fingerprinted as a Brave browser running Canvas Blocker which is way less common than just a Brave browser.

**Comment by dm17** [comment]
The reason the discussions "are never going to end" is because the people who run PTIO have political alliance that stipulates certain recommendations not be altered by these discussions. Very few have the balls to say it publicly, but what are they going to do? They're cowards.

**Comment by Shifterovich** [comment]
Since you're not a coward please try having the balls to back your claims with facts.

**Comment by dm17** [comment]
Sure, I have a whole thread about it: 
https://github.com/privacytoolsIO/privacytools.io/issues/856

**Comment by Shifterovich** [comment]
> the people who run PTIO have political alliance that stipulates certain recommendations not be altered by these discussions

> back your claims with facts

> #856

> ❌ Software Removal | Firefox

:thinking:

**Comment by five-c-d** [comment]
> can we make a final decision on Brave?
> 
> I personally see only one pro and one con.

The primary "pro" for BraveBrowser is that it is chromium-based, and thus, will function on sites that do not properly support TorBrowser (firefox-ESR-based) and Firefox-stable (timeshifted forward N weeks ahead of TorBrowser's baseline).  My anecdata says lots of sites are written for chrome, tested in chrome, and do not work to 'varying degrees' in things that are *not* chrome.  BraveBrowser is a kind of chrome, so it works with nearly all sites.  

It's not the best option, but it's the best chrome-a-like option nowadays, methinks.

**Comment by dm17** [comment]
> > can we make a final decision on Brave?
> > I personally see only one pro and one con.
> 
> The primary "pro" for BraveBrowser is that it is chromium-based, and thus, will function on sites that do not properly support TorBrowser (firefox-ESR-based) and Firefox-stable (timeshifted forward N weeks ahead of TorBrowser's baseline). My anecdata says lots of sites are written for chrome, tested in chrome, and do not work to 'varying degrees' in things that are _not_ chrome. BraveBrowser is a kind of chrome, so it works with nearly all sites.
> 
> It's not the best option, but it's the best chrome-a-like option nowadays, methinks.

This would actually be correct if the topic were ungoogled-chromium and not Brave.

**Comment by five-c-d** [comment]
I think we fully discussed the alternative-forks in 856.   But yeah, the case could be made that ungoogled-chromium is WorthMentioning... suggest you do that in #524 ...and once it *is* listed, then you can make the case for demoting BraveBrowser to worthMentioning, and promoting UngoogledChromium into top3 to replace it.  Finally, the culmination of your desire to remove all mention of braveBrowser, would be to get it demoted out of the WorthMentioning area, and delisted from privacyToolsIO entirely.  That would take several discussions and a lot of good solid reasons to make happen though.  

But as with your desire to take firefox out of the top3, removing it from the privacyToolsIO website entirely, and replace it with waterfox, something that is not yet even in the WorthMentioning listings ... 

I predict that your desire to take braveBrowser out of the top3, removing it from the privacyToolsIO website entirely, and replace it with ungoogledChromium, something that is not yet even in the WorthMentioning listings ... 

will probably not get very far.  Partly because I think those are the wrong tools for the target-audience readership, but mostly because you are arguing in a way that is counterproductive to your stated goals.  In your comment, you just, flatly assert, that I'm completely wrong -- but you provide no logic, no reasoned argumentation, no source-links, nothing except your assertion (and a complete requote of my entire comment-text ... please stop doing that).  

Such an approach is not the way you will convince people, that the drastic changes you advocate are the best way forwards.

**Comment by dm17** [comment]
> I think we fully discussed the alternative-forks in 856. But yeah, the case could be made that ungoogled-chromium is WorthMentioning... suggest you do that in #524 ...and once it _is_ listed, then you can make the case for demoting BraveBrowser to worthMentioning, and promoting UngoogledChromium into top3 to replace it. Finally, the culmination of your desire to remove all mention of braveBrowser, would be to get it demoted out of the WorthMentioning area, and delisted from privacyToolsIO entirely. That would take several discussions and a lot of good solid reasons to make happen though.
> 
> But as with your desire to take firefox out of the top3, removing it from the privacyToolsIO website entirely, and replace it with waterfox, something that is not yet even in the WorthMentioning listings ...
> 
> I predict that your desire to take braveBrowser out of the top3, removing it from the privacyToolsIO website entirely, and replace it with ungoogledChromium, something that is not yet even in the WorthMentioning listings ...
> 
> will probably not get very far. Partly because I think those are the wrong tools for the target-audience readership, but mostly because you are arguing in a way that is counterproductive to your stated goals. In your comment, you just, flatly assert, that I'm completely wrong -- but you provide no logic, no reasoned argumentation, no source-links, nothing except your assertion (and a complete requote of my entire comment-text ... please stop doing that).
> 
> Such an approach is not the way you will convince people, that the drastic changes you advocate are the best way forwards.

If you believe that ungoogled-chromium should be added, then why do you need my help to do it? I have no power or authority here. I have a feeling you're just trying to get me to waste my time.

**Comment by five-c-d** [comment]
I'm trying to get you to stop mass-quoting uselessly.  And stop attacking the people here that honestly disagree with you.  And start arguing with logic.  

But mostly I'm trying to help you see, that when you say "we should delete everything in the top3 and replace them with things which are not even in the worthMentioning section" that you are either 

1.  at the wrong website because you disagree with everything here 
2.  are advocating a drastic change for a very good reason AND NEED TO GIVE the very good reason by explaining what that good strong logical well-sourced reason *is*.  

Hint:  "dm17 is correct and 5cd is super-wrong" does not count as *reasoning* it only counts as bickering.  Hint2:  "dm17 is correct and anybody who disagrees is a shill" is even less worthy.  I know you CAN reason logically and argue coherently, all I'm asking is that you stick to doing that.  For instance here == https://github.com/privacytoolsIO/privacytools.io/issues/856#issuecomment-483488659  perfectly logical, albeit with the oversized-quotation.

**Comment by dm17** [comment]
> I'm trying to get you to stop mass-quoting uselessly. And stop attacking the people here that honestly disagree with you. And start arguing with logic.
> 

All baseless accusations. You've talked down to me since the beginning of my thread - which I argued without any mean tactics and with logic. Your authoritarianism is consistent regardless of the logic your opposition uses. It has been hashed and re-hashed in my previous thread.

> But mostly I'm trying to help you see, that when you say "we should delete everything in the top3 and replace them with things which are not even in the worthMentioning section" that you are either
> 

To help me see... Why are you quoting? That isn't a quote of me, so implying that it is is dishonest. 

> 1. at the wrong website because you disagree with everything here

I am not "at the wrong website."

> 2. are advocating a drastic change for a very good reason AND NEED TO GIVE the very good reason by explaining what that good strong logical well-sourced reason _is_.

Removing browsers that have betrayed user privacy is in line with what PTIO claims to be. You can capitalize all you want - it does not change the years of threads being ignored that oppose your views.

> 
> Hint: "dm17 is correct and 5cd is super-wrong" does not count as _reasoning_ it only counts as bickering. Hint2: "dm17 is correct and anybody who disagrees is a shill" is even less worthy. I know you CAN reason logically and argue coherently, all I'm asking is that you stick to doing that. For instance here == [#856 (comment)](https://github.com/privacytoolsIO/privacytools.io/issues/856#issuecomment-483488659) perfectly logical, albeit with the oversized-quotation.

Again, who are you quoting here? Your inner voice? I'm not going to submit to you. Silly.

**Comment by five-c-d** [comment]
I count several insults, several snarks, and ... perhaps unsurprisingly... zero additional logic.  

>  I argued without any mean tactics and with logic

You say you were logical.  But you are not adding any new logic.  Or even, rephrasing any existing logic.  You are just, claiming that ONLY you are being logical.  

>  the logic your opposition uses. It has been hashed and re-hashed in my previous thread

Again:  you assert you are logical.  But show no actual logic HERE in your comment.  

>  Removing browsers that have betrayed user privacy

Dramatic claim.  Back it up with logic.  Or with links.  Or at least with something more than bickering.  Assertions without reasoning to back them up, which would result in completely rewriting the what the website says now, would not be likely to get anywhere, logically speaking.  As well as historically speaking.  

>  the years of threads being ignored that oppose your views

This is not a vote, where whoever comments the most times, therefore wins.  If you want to just count noses, Chrome is the finest browser evah.  Neither of us are here to count noses.  The point of the discussions here, is to come up with good solid stable tool-recommendations that everyday endusers can use to boost their privacy *right now*, today.  You strongly dislike the current tool-recommendations and want to completely revamp them.  When you don't get your wishes satisfied, instead of stepping back and re-assessing, you assume that your opponents "have political alliance" or are "dishonest" or all your other attacks.  Zero of which are logical attacks.  Zero of which are truthful attacks.  

>  what PTIO claims to be

I just think you don't understand what it claims to be.  It says right on the tin what it claims to be.  It does not say "the website for tools that have developers who align with dm17's political stances".  It does have a political lean, but it is a pretty-non-partisan kind of political lean, and very technologically-oriented lean:  

*  Private and state-sponsored organizations 
are monitoring and recording your online activities. 
*  ...services, tools and knowledge 
to protect your privacy 
against global mass surveillance

The tools in the listings should help protect *the readership's* privacy, against *mass* surveillance.  If we want to help the masses, the tools have to be easy to use, the listings have to be stable, and the knowledge has to be something they can hope to attain.  The kind of tools is being defined as well:  software which works to keep surveillance-organizations from monitoring and recording the readership's online activities.  

>  No reason at this point that PTIO can't recommend 
>  ungoogled-chromium, waterfox, pale moon... 

Yeah... possibly?  Somebody would have to make the case, that those were WorthMentioning, and get them promoted from not-listed-at-all, to being in the listings.  

But you don't want that.  You want them to be the top three.  You want BraveBrowser and Firefox eliminated entirely from the listings, not just demoted from top3 into WorthMentioning but gone.  You've been given the reasons why you are wrong.  They were good reasons.  You didn't like the outcome.  So:  must be a conspiracy.  

>  I'm not going to submit to you

You are free to keep acting like you are acting.  But insulting everybody and demanding you get your way, is not going to end well for your goal of altering the listings.  My suggestion is simple:  take it step by step.  Try to get waterfox into WorthMentioning, since (to me at least) it is legitimately WorthMentioning.  

If you prefer tilting at windmills, continue, I guess.

**Comment by dm17** [comment]
> I count several insults, several snarks, and ... perhaps unsurprisingly... zero additional logic.
> 

Easy to say. Anyone who wants to know the truth in this category can read yours and mine posts on my long thread.

> > I argued without any mean tactics and with logic
> 
> You say you were logical. But you are not adding any new logic. Or even, rephrasing any existing logic. You are just, claiming that ONLY you are being logical.
> 

Lies. Lots of logical threads going back years, and I can't find a single one that the PTIO administration gave way to in terms of reducing their mainstream recommendations in the browser category. 

> > the logic your opposition uses. It has been hashed and re-hashed in my previous thread
> 
> Again: you assert you are logical. But show no actual logic HERE in your comment.
> 

Easy: lots of threads going back years with logic (summarized in those threads; I will not re-summarize) demonstrate that the PTIO folks ignore it. You permit the discussion and merely ignore it. So much showing that Brave and Firefox are not as privacy oriented as they claim.

> > Removing browsers that have betrayed user privacy
> 
> Dramatic claim. Back it up with logic. Or with links. Or at least with something more than bickering. Assertions without reasoning to back them up, which would result in completely rewriting the what the website says now, would not be likely to get anywhere, logically speaking. As well as historically speaking.
> 

Has all been linked in previous threads. You seem intent on getting me to repeat myself and wasting my time.

> > the years of threads being ignored that oppose your views
> 
> This is not a vote, where whoever comments the most times, therefore wins. If you want to just count noses, Chrome is the finest browser evah. Neither of us are here to count noses. The point of the discussions here, is to come up with good solid stable tool-recommendations that everyday endusers can use to boost their privacy _right now_, today. You strongly dislike the current tool-recommendations and want to completely revamp them. When you don't get your wishes satisfied, instead of stepping back and re-assessing, you assume that your opponents "have political alliance" or are "dishonest" or all your other attacks. Zero of which are logical attacks. Zero of which are truthful attacks.
> 

I understand that this is not a vote. Where did I claim it was?

> > what PTIO claims to be
> 
> I just think you don't understand what it claims to be. It says right on the tin what it claims to be. It does not say "the website for tools that have developers who align with dm17's political stances". It does have a political lean, but it is a pretty-non-partisan kind of political lean, and very technologically-oriented lean:
> 

Again making up quotes for me and using them as if they're read. Dishonest.

Anyway, I will continue to leave comments here if I so please - and you can continue misquoting me if you please. I'm not going to waste my time as much as I was previously answering everything point-by-point, however.

**Comment by five-c-d** [comment]
>  reducing their mainstream recommendations

Yes.  Because it is for everyday readership.  To help them boost their privacy, with tools that are useful for *that purpose specifically*.  

You are wanting the site to be aimed at a different kind of readership:  in particular, readership that is skillful enough to sha256sum ungoogledChromium and hand-update it if they have to.  Readership that is skillful enough to analyze and select a different browser in a few days, it the waterfox dev succumbs to the hit-by-a-bus factor.  

But more than that, you are wanting the listings to be politically determined, based on things related to what one of the developers donated to, or based on what some other developer-group removed a third-party thing from their website, and so on.  Things which have very little to do with the *tools* per se, things which are about the humans behind the tools. 

You ALSO have some concerns related to how much privacy the tools provide, which is good -- you ain't all bad  :-)   But you have a hard time separating your personal desires, from what is good for the target-audience, what tools will help them improve their privacy.  Whether or not gabDissenter is or is not in the plugin-area, is not a privacy-issue.  Whether or not some developer did or did not donate to a political cause, is not a privacy-issue.  They are not even issues about the TOOLS which are what the listings are about.  

>  I understand that this is not a vote. Where did I claim it was?

Everything you say is complaining about *your* voice, the *number* of ignored threads.  

You don't link to a logical point which was missed, or unaddressed.  You don't bring up new information, something that has changed which would potentially alter the equation.  You just say "lots of" ...and think the vote-tally matters!  It does not matter what the tally was, because this is not a political election of developer-politicians.  The tools are tools.  They are listed if they are tools that help everyday people get privacy.  They are delisted if something is better *for that specific purpose* comes along.  But it doesn't get rewritten overnight usually, because doing that without a REALLY good reason, wouldn't serve the interests of the readership -- everyday endusers that want solid stable long-haul recommendations.  

>  Lots of logical threads going back years, and I can't find a single one

You see this as a vote:  "lots".  It is not a vote:  "a single one".  You disliking the outcome of past discussions, does not make those decisions wrong.  You liking certain tools, which are not suitable for the top3 on *this* website because the tools you like are not suitable for the target-audience, is up to you.  You wanting to see everything in the listings rewritten, though, is only something you can accomplish by seeing clearly, not by seeing conspiracy.

**Comment by dm17** [comment]
> You are wanting the site to be aimed at a different kind of readership: in particular, readership that is skillful enough to sha256sum ungoogledChromium and hand-update it if they have to. Readership that is skillful enough to analyze and select a different browser in a few days, it the waterfox dev succumbs to the hit-by-a-bus factor.
> 

You have demonstrated you have no idea what I want... Or you know and you're using nasty tactics to get me to waste my time. The way you talk to people is insane.

**Comment by five-c-d** [comment]
What you want is clear.  

*  You want Firefox blacklisted.  You want Waterfox in the top3.  
*  You want BraveBrowser blacklisted.  You want UngoogledChromium in the top3.  

When you didn't get your way, you decided that the REAL reason you didn't get your way, was because the people running this website are evil.  As you seem to do with anybody who disagrees with you on any points.  Even when, like me, they agree with you on several points.  

In particular, you do that stuff, as opposed to, stepping back, and assesssing whether you just didn't understand the purpose of the website.  Or whether you might be wrong.  Or whether your habit of intermixing the personal, the political, the conspiratorial, and the actually-related-to-the-privacy-protecting-properties-of-the-software-tools ... whether that is an approach likely to bring you success in your desire to dramatically alter the listings.  

BraveBrowser is not perfection incarnate.  But it's a big step up from Chrome.  

The majority of the readership -- because the target-audience is everyday humanity -- are using Chrome right now.  They would be better of using Firefox, or TorBrowser, or BraveBrowser.  You disagree.  You think that is not enough, because you have other criteria besides improving the toolset (and how it helps their privacy specifically) of everyday readership.  You also want to have politically-motivated criteria, not because you think it matters to the everyday readership -- just, because those political things matter to you.

**Comment by dm17** [comment]
> What you want is clear.
> 
> * You want Firefox blacklisted.  You want Waterfox in the top3.
> * You want BraveBrowser blacklisted.  You want UngoogledChromium in the top3.
> 

No, I want PTIO to do more than pretend to listen to evidence that is presented what it goes against their view of promoting mainstream browsers (that deliberately whitelist Facebook and Twitter trackers, for example in Brave).

Blacklisted? They can be mentioned, but I do think the most private browsers should be congruent with the evidence. I would say Pale Moon over Waterfox, actually.

**Comment by five-c-d** [comment]
> the most private browsers

This is not the goal of the listings.  That is what I mean when I say you don't understand the site, and you are trying to change who the readership is.  Where does it say "THE MOST private" anywhere on the site?  

The everyday readership is not technologically advanced enough to use ELinks on air-gapped OpenBSD, despite the immunity from JPEG-parsing zero-days it could theoretically give them.  I'm sure you will complain that Pale Moon is *so* much more advanced that ELinks, but that is missing the point.  Pale Moon is not suited to the target-audience.  

See the 856 discussion, but instead of reading it with the mindset that there is a conspiracy to keep establishment-players BigMozilla and BigTor and BigBrave dominant over heroic grassroots champions WaterFox and PaleMoon and UngoogledChromium... try reading it with the mindset of: 

*  "can the normal average everyday person 
who uses chrome with default settings 
on windows 10 with default settings 
as their current toolset right now today 
benefit one hour later from these listings 
and continue for the next three years 
enjoying significantly improved privacy 
WITHOUT spending endless hours troubleshooting 
and/or recovering from security incidents".   

With waterfox the chances are probably 50/50, but the other two, *way* lower.  With Brave the chances are probably 80/20, with Tor 90/10, and with Firefox 95/5 or 90/10 depending on whether the fall off the deep end.  

> deliberately whitelist Facebook and Twitter trackers

Is this good for privacy, of the target-audience?  Depends on whether they alter the setting.  And depends on whether, if they installed something else, it would just make them run screaming back to Chrome... which is the opposite of the goal here, the listings have to be aimed at the target-audience.  That means, NOT picking "the most private X" but something very very different:  picking the X which will significantly boost privacy, which is easy enough for the mere mortal to install, which is easy enough for the mere mortal to use, which is long-haul enough they won't go back to chrome because of tool-churn / hit-by-a-bus-factor / etc, and which in general is going to help fight mass surveillance.  

So in addition to not just having a contest to see what "the most private X" is amongst a particular tool-category, but also taking other concerns into consideration, there is a firm limit on what other concerns ARE relevant.  What political party the tool-developer belongs to, what causes they donate unto, what things they say on their property, is not relevant to the goal of fighting global mass surveillance-organizations.   It's important, don't get me wrong, to fight censorship and such... but this is not the place to fight those battles.  This is the place to concentrate on the privacy provided by the tools AS tools, to the everyday readership.  You need the masses to actually USE the tools, if you want to fight mass surveillance.

**Comment by dm17** [comment]
>The everyday readership is not technologically advanced enough to use ELinks on air-gapped OpenBSD, 

Exactly why I'm not going to waste my time talking to you anymore. Every time someone makes a valid argument for a more secure browser you just exaggerate and try to make them seem silly.

**Comment by five-c-d** [comment]
> more secure browser

This is not the goal of the site.  The point of the exaggeration is not to make you seem silly, the point is to make you understand that there are different potential goals.  If the goal was "the most private" then Waterfox and UngoogledChromium and PaleMoon **would lose** because they are not TheMostPrivate and *definitely* not TheMostSecure.  

If you want to make valid arguments, you are welcome to.  But you've made the point that BraveBrowser whitelists twitter a bunch of times.  

*  It is simply unfair to be promoting Brave & Firefox as the top privacy browsers. Brave defaultly whitelists Facebook and Twitter trackers  

*  Brave - how can defaultly whitelisting Facebook & Twitter trackers be seen as 'privacy centric?'

*  that deliberately whitelist Facebook and Twitter trackers, for example in Brave

You are going to keep making that point again and again, until somebody does what you want and moves your preferred tools into the top3, is apparently your strategy.  

But what is your beef here?  Well, it is very simple.  Like usual, you have a *political* beef.  

*  [Twitter censors](https://github.com/privacytoolsIO/privacytools.io/issues/856#issue-432659787) everyone, so we have Gab... and Mozilla censors it

That has nothing to do with whether firefox, the tool, is going to boost the everyday enduser's privacy more than continuing to use Chrome.  

The reason I'm pointing out the flaws in your arguments, is because I want the listings to be suitable for the intended target-audience.  The reason you have a flawed argument, is because you want simply repeating things you have said before, to suddenly cause everybody to follow your lead.  But more fundamentally, the reason you assume everybody is conspiring against you, is because you just don't want to recognize that the site has a specific target-audience, and that your recommendations are unsuited to that audience, except perhaps in the WorthMentioning area (or the oft-mentioned but never yet implemented "for advanced endusers willing to go the extra mile" section which DOES concentrate on listing tools which are closer to "the most private X" as opposed to tools which everyday endusers can handle)

**Comment by dm17** [comment]
There's nothing wrong with starting a privacy-focused website to popularize and recommend specific privacy softwares. You're welcome to do it. I cannot remove your website from the internet, and I've not implied anyone should. However, the messaging does not match the recommendations. Firefox & Brave simply aren't "privacy browsers" anymore. It is quite easy; you don't need to run OpenBSD or even do any custom configuration (which your website admits needs to be done with Firefox). All a novice user needs to do is install something like Pale Moon (or similar for unsupported platforms). 

>If the goal was "the most private" then Waterfox and UngoogledChromium and PaleMoon would lose because they are not TheMostPrivate and definitely not TheMostSecure.

What do you think is the most private, then? "Suitable for target audience" is defined by what? Many of these browsers are just as easy to install and have automatic updates. And Firefox needs to be customized in about:settings - so your supposed criteria don't even match your recommendations.

**Comment by five-c-d** [comment]
> "Suitable for target audience" is defined by what?

I'm not sure which you mean.  Who defines the target-audience?  The site-owners do.  Who defines the suitability-criteria that the target-audience desires?  Ideally it would be the people *in* the target audience, but in practice, it is the site-owners trying to gauge (from their common sense and from the forums and from the webserver access logs and such) what makes sense and what does not make sense.  

The purpose of the site is not listing "the most secure X" nor is it listing "the most private X" ...the purpose is, tools & knowledge that help protect the masses, from mass-surveillance-orgs, online.  

>  What do you think is the most private, then?

It's not the purpose of the site, so I'll skip.  I do have some opinions on this tho  :-)  

> the messaging does not match the recommendations. 
>  Firefox & Brave simply aren't "privacy browsers" anymore

That is your position, correct.  But to me, the messaging is clear, you just misunderstand what the messaging means.  Perhaps I'm wrong, and you are correct.  Here is what the messaging is:  

*    Private and state-sponsored organizations
*    are monitoring and recording your online activities.
*    ...services, tools and knowledge
*    to protect your privacy
*    against global mass surveillance

You think listing Firefox + firefoxEsrBasedTorBrowser + BraveBrowser, violates that mission-statement somehow.  But since this is the BraveBrowser thread, just stick with explaining how BraveBrowser violates it, and why [some other chromium-based browser] would be better FOR THE READERSHIP specifically, to satisfy the stated mission specifically.  

Not how it would be "the most private X" compared to Brave, but how it would, for a normal human running Chrome, be more likely to improve their privacy overall, during the next three years, if they picked [your choice] rather than if they picked BraveBrowser.  You have noted one specific actual downside:  

1.  BraveBrowser whitelists Facebook by default 

The solution is to alter the settings (more knowledge) or install a plugin/addon (more tools).  Downside:  note everybody will figure that out, the first time through.  What are the ***other*** downsides, to BraveBrowser, besides that one you keep mentioning?  Then, give the pros and cons of your proposed alterative.  Is your proposed alternative, already in the WorthMentioning?  If not... why not?  Shouldn't that be *the starting point* of the discussion, rather than, oh we can just delist Brave and bring in something not-even-listed-at-all straight into the top3... ?

**Comment by dm17** [comment]
Right, so you don't believe "controlled opposition" exists - despite the fact that it has been written about in business & war literature for centuries. Firefox & Chrome are like Republican and Democrat. They're both huge and untrustworthy, and there's plenty of data on the web showing this. So unless the target audience is, "dummies who want to be scammed into thinking they're getting what the private browser market has to offer," then these are still dishonest recommendations.

Nothing hard about linking to easy-to-download browsers like Waterfox & Pale Moon. And nothing hard about downloading them as a "novice" or "general audience."

**Comment by dm17** [comment]
Interesting watching you rationalize back and forth between advanced users not being your target audience and entering all sorts of browser customizations in to disable privacy-violating features.

**Comment by five-c-d** [comment]
I think the site has two target-audiences, and needs to be in two sections.  So does the founder, but it is slow going to actually implement the idea (it has been around for a long time apparently).  But you are also assuming, incorrectly and insultingly-as-per-usual, that I personally think the listings are 100% perfect.  I definitely do not.  

As for whether controlled opposition exists, sure... but you are asserting, in your conspiracy theory, that the core team of privacyToolsIO, ***is*** the controlled opposition, aka evil people out to hoodwink the world.  Which is complete bollocks.  

>  Firefox... untrustworthy, and there's plenty of data on the web showing this

Link me to the best most well-researched most logically-reasoned three places that show this.  Or how about, even just one link for starters.  Something you haven't already linked to please, I've read 856.  

> nothing hard about downloading them as a "novice"

I notice you are back to talking about Firefox-forks again.  And I notice you are specifically saying "oh you can click the link and get the executable easily" which is completely different from what I actually asked.  Namely, what the chances are that the person who DOES take your advice, will be satisfied with your recommendation for several years, will not need to spend hours troubleshooting, will not be burned by security incidents, and will not be sent screaming back to chrome.  Yes, anybody can download UngoogledChromium, kind of.

**Comment by KonoromiHimaries** [comment]
>  I would say Pale Moon over Waterfox, actually.

@dm17 i recommend Basilisk Browser instead Pale Moon
https://github.com/MoonchildProductions/UXP

**Comment by KonoromiHimaries** [comment]
> Link me to the best most well-researched most logically-reasoned three places that show this. Or how about, even just one link for starters. Something you haven't already linked to please, I've read 856.

https://github.com/ghacksuserjs/ghacks-user.js/

**Comment by five-c-d** [comment]
Okay?  That's mentioned in the listings.  As something that advanced users might want to do, when installing firefox.  How is that relevant to removing BraveBrowser from the top3, and replacing it with "other thing"?

**Comment by dm17** [comment]
>As for whether controlled opposition exists, sure... but you are asserting, in your conspiracy theory, that the core team of privacyToolsIO, is the controlled opposition, aka evil people out to hoodwink the world. Which is complete bollocks.

No, I'm asserting they're acting like it, not that they are. Acting like it is ignoring so many threads & evidence to the contrary over the years while they present themselves as subject-matter experts. That's a mismatch.

>will not be burned by security incidents

No evidence that Pale Moon or Waterfox users were burned by security incidents.

> Or how about, even just one link for starters

https://spyware.neocities.org/articles/firefox.html

You can call it "political," but that doesn't discredit it at all - stuff like this matters:
https://www.activistpost.com/2017/08/mozilla-joins-george-soross-efforts-launching-strike-fake-news.html (lots of other references for this if this one irks you for some reason)

Novice users should have to do this?
https://www.askvg.com/tip-disable-telemetry-and-data-collection-in-mozilla-firefox-quantum/
More explanation why they'd want to:
https://www.reddit.com/r/privacy/comments/9jvxxe/what_is_wrong_with_browser_telemetry/
If a novice ends up searching for how to turn it off, then they'll likely find this at the top:
https://support.mozilla.org/gl/questions/1197144
Doesn't seem exactly straightforward for a novice or "target audience."

**Comment by five-c-d** [comment]
Okay, we're getting closer.  Now, if we can shift gears back to BraveBrowser, and keep the same approach, we might get somewhere.  

> You can call it "political," but that doesn't discredit it at all - stuff like this matters

Never said it didn't matter.  Just said, it is not something that matters with respect to helping everyday endusers get educated about what tools will help them protect themselves from global mass-surveillance entities.

**Comment by dm17** [comment]
> Okay, we're getting closer. Now, if we can shift gears back to BraveBrowser, and keep the same approach, we might get somewhere.
> 
> > You can call it "political," but that doesn't discredit it at all - stuff like this matters
> 
> Never said it didn't matter. Just said, it is not something that matters with respect to helping everyday endusers get educated about what tools will help them protect themselves from global mass-surveillance entities.

What about Brave? This thread is already closed, so I don't see how this is not a waste of time. Brave was launched by the very intelligent Eich, who did a scam of donating a minute $1K against gay marriage while being obviously gay himself. Not sure why so many fell for this silly plot to get him his own browser wherein the ad revenue stream is effectively usurped. Why isn't it obvious to you that Brave has shifted the incentive from the advertisers to itself for collecting data and targeting users? This is self evident in terms of the Brave-ad model.

**Comment by five-c-d** [comment]
>  I don't see how this is not a waste of time

You keep saying that word.  

To me, about 70% of what we're doing here is trying to regain some friendliness so that it is not such a pain when we discuss in future threads.  I expect this is not the last time we will discuss browsers.  

But I also think we are not wasting time, discussing whether BraveBrowser belongs, **if** we are documenting actual facts rather than just bickering.  That an issue is "closed" has almost zero bearing on anything; that is not some kind of final determination.  Issues can always be reopened.  

> Brave has shifted the incentive from the advertisers 
> to itself for collecting data and targeting users?

By comparison to firefox?  By comparison to ungoogledChromium?  By comparison to GoogleChromeRunningOnWindowsTen?  

BraveBrowser has a specific bunch of claims on data-collection.  Which you've read, but found laughable (yet did not comment on why their reasoning is incorrect).  Here == https://github.com/privacytoolsIO/privacytools.io/issues/161#issuecomment-296416663 

To me, this is a place where BraveBrowser is not a good as MozillaFoundation and TorFoundation ...but I have no illusions about Foundations always being superior, there are a lot of ways that can go wrong (such as spending money and headcount on political things rather than on improving the browser-codebase).  Firefox has a "business model" even if they are technically a non-profit, which is why they have EME built in, and why they have google as the default search engine, and so on.  BraveBrowser does most of the same things, and intends to monetize the advertising end somewhat in a similar fashion to the way that Google-of-1998 wanted to operate:  algorithmically, not putting a thumb on the scale.  Sure, things might go bad on BraveBrowser's parent-entity, the same way GoogleSearch's parent-entity gradually (but increasingly quickly) lost their way on a large number of issues.  Sure it would be better if the adblocker in braveBrowser was not morphing into an adRevenueTransfer type of thing.  

But this is not about "what is the most private X" this is about, what are the alternatives that the everyday readership might be able to cope with?  Is Chrome better for their privacy, than BraveBrowser?  Definitely not, and it isn't even close.  Is ungoogledChromium better for their privacy than BraveBrowser?  Maybe... but it strongly depends on how tech-savvy the enduser in question is, right?  Teamsize very small, longetivity uncertain, patch-cadence not as speedy, etc.  Will there still be a BraveBrowser in 2022, and will it still be a decent amount of privacy for an everyday enduser, without much hassle/risk, between now and then?  Yes, probably 70% chance.  Will there still be an UngoogledChromium in 2022, and will it still be a decent amount of privacy for an everyday enduser, without much hassle/risk, between now and then?  Maybe, but the chances are considerably worse, because of small teamsize, slower patch-cadence, uncertain trademark risks, etc.  I also think the privacy-differential between BraveBrowser-with-some-tweaks, and UngoogledChromium-with-some-tweaks, is pretty tiny compared to the difference between TorBrowser and BraveBrowser, for instance.

**Comment by dm17** [comment]
There will obviously need to be prioritization between "how long will the browser exist" and "how much does the browser reduce privacy." PTIO implies the prior is much less important than the latter - doesn't it?

**Comment by five-c-d** [comment]
It is a hard question to answer.  It depends on whether you think fighting global mass-surveillance-entities, is a game which can be won.  To me it is more like an ongoing struggle, and therefore, I don't want something that might disappear a couple years later.  Waterfox seems to pass that test, albeit with some risk because of the centrality of the key developer on the project.  PaleMoon/Basilisk/etc, definitely not.  UngoogledChromium, probably not but a bit tougher of a call.  BraveBrowser, very likely survives.  TorBrowser, almost certainly survives.  

It always helps my thinking when I split into everyday-endusers versus advanced endusers willing to go the extra mile though:  firefox+ghacks is only for advanced users... in which case, they don't need waterfox.  TorBrowser with noscript is for advanced users, and I would say is flat out better than firefox+ghacks, but it depends on if anonymity is more valuable than adblocking to the advanced enduser in question.  But when I think about normal endusers, ghacks is out of the question, and chromium-based-engine more critical, and longetivity/stability of the software FAR more important.  Same with patch-cadence, it is not as critical for advanced endusers that know that they are doing, as it is for mainstream folks.

**Comment by Shifterovich** [comment]
> It's not the best option, but it's the best chrome-a-like option nowadays, methinks.

How though? It's more fingerprintable than Chrome.

**Comment by five-c-d** [comment]
Privacy stuff:  

* main search engine is DuckDuckGo, called to boycott Google/Gmail/Chrome/etc, [anti-phone-home](https://github.com/privacytoolsIO/privacytools.io/issues/649#issuecomment-445482505) checker 
* support for [Tor tabs](https://www.ghacks.net/2018/06/29/brave-browser-gets-private-tab-with-tor-option/) in private-browsing-mode 
* off-by-default (but built-in) NoScript 
* blocks ads, on-by-default 
* blocks ([most](https://www.reddit.com/r/privacy/comments/ap8rnv/brave_privacy_browser_is_whitelisting_trackers_of/eg74c8e/) but [not all](https://nakedsecurity.sophos.com/2019/02/12/privacy-browser-braves-user-concern-over-facebook-whitelist/)) website trackers & third-party cookies 
* httpsEverywhere to minimize tracking by the ISP 
* good security and stability/uptime (now more tightly aligned with chromium and has a speedy patch-cadence and a teamsize sufficient to keep that up) 

Tossup stuff:  

* some attempt to give fingerprinting protections (screensize/fonts/etc), will improve as userbase increases & codebase matures 
* some really interesting efforts to get verifiable builds + SGX enclaves that allow [remote attestation](https://www.reddit.com/r/privacy/comments/ap9149/brave_privacy_browser_has_a_backdoor_to_remotely/eg7auye/) 
* there are problems with the webRTC leak thing (e.g. thwarts VPN service), https://restoreprivacy.com/secure-browser/ has some links about trying to plug the leak, cf /r/privacyToolsIO [comment](https://www.reddit.com/r/privacytoolsIO/comments/a6l3lo/brave_vs_firefox_data_privacy/ebw22p2/)
* not as anonymized as TorBrowser 
* not as easy to max out (in the hands of an expert) as firefox 
* controversial plans to use targeted advertising by locally analyzing the anonymized browsing history of endusers
* controversial pay-to-surf plan where endusers can get bespoke cryptocurrency by viewing adverts (BraveCorp also gets some and advert-publisher gets some) 
* controversial idea for opt-in advert-replacements sold by BraveCorp, in place of adverts blocked by BraveBrowser 
* controversial [header injection](https://github.com/privacytoolsIO/privacytools.io/pull/657#issuecomment-463314974)

Usability stuff:  

* good performance e.g. pageload speed 
* properly cross-platform (e.g. better than TorBrowser) 
* good engine-compatibility with extant and future websites, good extensions-compatibility too, because engine is directly chromium-based (["I liked where Firefox was going, however, I couldn’t stick with it over the long term. It wasn’t compatible..."](https://www.theverge.com/2019/3/4/18249623/brave-browser-choice-chrome-vivaldi-replacement-chromium))
* good UI-compatibility with people that are used to how chrome4windows does things ("[zero learning curve](https://www.fastcompany.com/90321235/i-protected-my-privacy-by-ditching-chrome-for-brave-and-so-should-you)") 

This is my couple-hours-of-peeking-around list.  Should not be considered definitive.  But I agree for the most part with the tooltip on the current https://privacyTools.io/browsers listing:  "if you want a chromium-based browser then pick brave, though be aware is not as good as max'd-out-firefox"  

More to the point of this thread though, I don't think that UngoogledChromium is a suitable replacement in the top3 -- not aimed at everyday endusers.  No opinion on #973 which is IridiumBrowser ... never heard of it.  And by default, any browser not well-documented enough to make this list, https://en.wikipedia.org/wiki/Comparison_of_web_browsers , should not be in the WorthMentioning on privacyToolsIO -- let alone the top3.  The pickings are slim for libre-licensed Blink browsers; there are some webkit-ones like GnomeWeb-fka-Epiphany and KdeFalkon-fka-Qupzilla from the firefox-is-too-bloated factions in linux-land, but that is it.  

Thus, like I say, my analysis is that Brave is the best chrome-a-like at the moment.  It has pretty sane settings for normal everyday endusers, out of the box giving them better privacy than they have with chrome+windows10+bareIP+gmail ... if they switch to brave+qubes+mullvad+tutanota that is a significant upgrade in privacy.  The won't likely run screaming back to their bad old ways, either.  It works on all their devices they use now, with all their websites they use now, minimal hassles for a modicum of privacy.

**Comment by Mikaela** [comment]
I propose locking this old thread

**Comment by five-c-d** [comment]
> locking this

Or you could, just ask that we please open a new thread, if there is anything left to say  :-)  

*  There is already an open thread about adding IridiumBrowser to worthMentioning, 
*  and I'm considering opening one about Waterfox ... probably a "discussion whether it is worth mentioning" rather than a suggestion it ought to be though, since I do not use it myself.  
*  I think that ungoogledChromium was discussed in 856 sufficiently, but if somebody wants to open a new thread about whether it is worthMentioning that has not yet happened == https://github.com/privacytoolsIO/privacytools.io/issues?utf8=%E2%9C%93&q=UNGOOGLEDCHROMIUM 

BraveBrowser is in the listings, if somebody has new information though why it ought to be removed (that hasn't already been discussed upthread here), should they open a new thread rather than reviving a closed older issue?  Actual question, since I don't know what privacyToolsIO core team prefers in such a situation.

**Comment by Mikaela** [comment]
>  should they open a new thread rather than reviving a closed older issue? Actual question, since I don't know what privacyToolsIO core team prefers in such a situation.

I can only say that personally I would prefer a new issue rather than dozens of very long comments after issue is closed and I didn't see anything new appearing in those comments (or it was lost to their length). And now that this issue was actually locked, new issue would be preferable if there are new _very heavy arguments_ or if there is more to say I think [our forums would be even more preferably place](https://forum.privacytools.io/).
